tests/pauli/test_grouping_utils.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
```

Comment:
Nothing in this file is new (except for the module naming); these are just tests from the original `grouping` module moved into a new file.